vaadin-cookie-consent-flow-parent/vaadin-cookie-consent-flow-demo/src/main/java/com/vaadin/flow/component/cookieconsent/CookieConsentView.java

```diff
@@ -50,7 +50,7 @@ private void createCard(String heading, CookieConsent cookieConsent) {
         button.addClickListener(e -> {
             clearCookieConsent();
             this.cookieConsent = cookieConsent;
-            add(cookieConsent);
+            this.getElement().appendChild(cookieConsent.getElement());
```

Comment:
Can you elaborate what's going on here? The custom JS below already looks weird - is this really our API for showing the consent dialog or is this specific to the demo setup?